<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fereastrastergeAmenzi</class>
 <widget class="QDialog" name="fereastrastergeAmenzi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#fereastrastergeAmenzi{
background:url(:/imaginebuna.png)
}</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>10</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Inapoi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>440</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>STERGE</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>380</y>
     <width>113</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Introdu ID-ul amenzii:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelInformatie">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>410</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Afiseaza</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
